src/styles/_utility.css

```diff
@@ -1,7 +1,9 @@
-.hidden {
-  display: none !important;
-}
-
 .m0 {
   margin: 0 !important;
 }
+
+.df {
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+}
```

Comment:
I think this does a bit too much for a utility class. I guess `df` stands for `display: flex`? If so, I would leave out the `list-style-type` as it has nothing to do with a flexbox.